src/manifold/src/sort.cpp

```diff
@@ -464,6 +464,7 @@ void Manifold::Impl::GatherFaces(const Impl& old, const Vec<int>& faceNew2Old) {
     gather(policy, faceNew2Old.begin(), faceNew2Old.end(),
            old.meshRelation_.triProperties.begin(),
            meshRelation_.triProperties.begin());
+    meshRelation_.numProp = old.meshRelation_.numProp;
```

Comment:
It looks like you may have started with your other PR's branch. Should probably rebase onto master to avoid merge conflicts - and to make it easier to review.